apps/hash-graph/lib/graph/src/store/postgres/ontology/ontology_id.rs

```diff
@@ -5,15 +5,12 @@ use serde::{Deserialize, Serialize};
 use utoipa::ToSchema;
 use uuid::Uuid;
 
-// TODO: rename this to TypeInternalVersionId or something to distinguish it from versioned
-//  URIs and from entity ids
-//  https://app.asana.com/0/1202805690238892/1203214689883089/f
```

Comment:
I didn't even know we had this task when you talked about doing this. Great thing we already wanted to change this!

Reply:
Me neither, I found the TODO and was happy to remove it 😆